src/components/calculator.jsx

```diff
@@ -18,11 +18,14 @@ function Calculator() {
   };
   return (
     <>
-      <div className="calculator">
-        <OutputScreen state={current} />
-        <div className="keys">
-          <OperationKeys clickEvent={clickEvent} />
-          <NumericKeys clickEvent={clickEvent} />
+      <div className="section">
+        <h2>Lets Do some Math</h2>
+        <div className="calculator">
+          <OutputScreen state={current} />
+          <div className="keys">
+            <OperationKeys clickEvent={clickEvent} />
+            <NumericKeys clickEvent={clickEvent} />
+          </div>
         </div>
```

Comment:
- [ ] Again here kindly align the calculator UI so as to match the given wireframe. Notice how the calculator UI aligns with the `links` at the top to the right.

Your design
![image](https://github.com/misalazeem/Math-Magicians/assets/81932478/9ff07fa7-02aa-48d8-98f4-81f508147ba4)

Given wireframe
![image](https://github.com/misalazeem/Math-Magicians/assets/81932478/89a96ef1-5678-43b4-92a1-824f5b8e6310)